mautic/mautic
# [UI] Replace poor icon size solution by classes

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This pull request addresses the issue of inconsistent icon sizing in the UI, which previously relied on a generic CSS rule applied to table elements. By replacing this approach with specific classes in the Twig structure, we ensure a more flexible and maintainable solution for adjusting icon sizes.






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the Forms page, create a Campaign Form, save it, and then close it. Go back to the list and verify that the Campaign icon is displayed. Use your developer tools to inspect the icon and ensure its size is 14px.
